components/inputs/input-time-range.js

```diff
@@ -160,17 +160,21 @@ class InputTimeRange extends FormElementMixin(RtlMixin(LocalizeCoreElement(LitEl
 		const startLabel = this.startLabel ? this.startLabel : this.localize('components.input-time-range.startTime');
 		const endLabel = this.endLabel ? this.endLabel : this.localize('components.input-time-range.endTime');
 
+		const startTimeInput = this.shadowRoot.querySelector('.d2l-input-time-range-start');
```

Comment:
Start and end wont really have their own errors - they can never be empty so they wont have an error state if empty and required. The only error possibility for times is if start time is after end time in the parent (though maybe this change is necessary still)

Reply:
I don't think this is technically needed. I wrote it this way so that if `input-time` gets modified to support some sort of error, `input-time-range` will automatically handle it correctly.